tests/julia/julia_version-1.1/README.rst

```diff
@@ -1,5 +1,4 @@
-Julia - REQUIRE: test version for julia v1.1
+Julia - Project.toml: test version for julia v1.1
 ---------------
 
-Starting with julia v0.7 and up, the package manager has changed, so this tests
-that the julia version can be specified and packages installed correctly.
+Test a Project.toml file for julia.
```

Comment:
Related to the comment earlier ("add a test to select non default version"). Should we rename this test so it conveys the idea "this test checks that the build pack selects the default version of Julia for a Project.toml based repo". Currently that default version is 1.1 but at some point it will get bumped up (I guess).

Then with a second test that specifically selects a version that isn't the default (say 1.0?) we'd cover both options.

Reply:
Done!